app-modules/integration-open-ai/config/integration-open-ai.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+
+    'gpt_35_base_uri' => env('OPEN_AI_GPT_35_BASE_URI'),
+
+    'gpt_35_api_key' => env('OPEN_AI_GPT_35_API_KEY'),
+
+    'gpt_35_api_version' => env('OPEN_AI_GPT_35_API_VERSION'),
```

Comment:
Per our discussion with Joe earlier today, this will need to be changed to be per Tenant stored in a setting or something similar. But it can be left for now.